### PR TITLE
login page ui + integration and logout

### PR DESCRIPTION
**Pull request by sekarminati**

### CODE REVIEW

File: src/pages/Login.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  const onFinish = (values) => {
+    axios.post(config.apiHost + "/v2/login", {
+      username : values.username,
+      password : values.password,
+    })
+    .then(function(res){
+      // success
+      window.localStorage.setItem("token", res.data.data.token)
+      history.push('/admin/dashboard')
+      console.log(res.data.data.token)
```
Comment (vicimilenia) [inline src/pages/Login.js]: please delete if not needed :D

File: src/pages/Login.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    })
+    .catch((err) => {
+      //error
+      message.error("Username and password doesn't match");
+      console.log(err, "error");
```
Comment (vicimilenia) [inline src/pages/Login.js]: please delete if not needed :D

File: src/pages/Login.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    })
+  };
+      
+  const onFinishFailed = errorInfo => {
+    console.log('Failed:', errorInfo);
```
Comment (vicimilenia) [inline src/pages/Login.js]: please delete if not needed :D or maybe change to 
```suggestion
   message.error(errorInfo);
```

**Review by vicimilenia — CHANGES_REQUESTED** [review]

**Review by sayaandreas — APPROVED** [review]
LGTM

**Pull request merged**